fastly/client.go

```diff
@@ -404,6 +404,8 @@ type RequestOptions struct {
 	Body io.Reader
 	// BodyLength is the final size of the Body.
 	BodyLength int64
+	// Context is a context.Context object that will be set to the Request's context.
```

Comment:
```suggestion
	// Context, if supplied, will be used as the Request's context.
```

No need to repeat the type in the doc comment :-)

Reply:
This change should be made in all the affected files so they are consistent.

As an alternative, consider creating a `RequestContext` struct in `client.go` which just contains this field, and can be embedded into all the `Input` structs. That way there's no need for a doc comment in the `Input` structs at all.

Reply:
fixed in `c2cbbd48048ef952961b79031e01ddaa15e44ea2`